lms/static/sass/_build-course.scss

```diff
@@ -30,7 +30,6 @@
 @import 'course/modules/student-notes'; // student notes
 @import 'course/modules/calculator'; // calculator utility
 @import 'course/modules/timer'; // timer
-@import 'course/modules/chat'; // chat utility
```

Comment:
Yeah - I'm guessing we don't have a way to dynamically pull in assets conditionally on whether or not you're using the feature. Seems like kind of a hard problem. @singingwolfboy any of your work addressing this?